docs/api.rst

```diff
@@ -3696,6 +3696,19 @@ InteractionMessage
 .. autoclass:: InteractionMessage()
     :members:
 
+    .. automethod:: InteractionMessage.delete
+
+    .. automethod:: InteractionMessage.edit
```

Comment:
Should fetch also be documented?

Given it does exist, although it'd fetch this object so not sure.

Reply:
That is for InteractionMessage, not PartialInteractionMessage.

For PartialInteractionMessage, everything is documented